github/resource_github_team.go

```diff
@@ -5,13 +5,25 @@ import (
 	"log"
 	"net/http"
 	"strconv"
+	"time"
 
 	"github.com/google/go-github/v39/github"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/customdiff"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
 	"github.com/shurcooL/githubv4"
 )
 
+/*
+These constants are used to retry API on various operations.
```

Comment:
In other words, we're trying to account for race conditions...
Ideally, the pending batch of changes would be executed in a top-down approach, where the right order of operations would be respected, w/o the need to hammer the API, hoping for the state we expect.

Reply:
We cannot guarantee the correct order of operation as terraform by default runs 10 parallel instances.

Reply:
Aye, but the resources within the provider should have explicit dependencies on one another.
For example, TF needs to be smart enough to know that it cannot create a user unless it first created the team that the user belongs to.